app/views/content/life-as-a-teacher/pay-and-benefits.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Pay and benefits"
+heading: "Pay and benefits"
+category: Life as a teacher
+image: "static/images/content/hero-images/0029.jpg"
+title_paragraph: Find out about all the benefits that come with a career in teaching. 
+description: |-
+  Find out about all the benefits that come with a career in teaching, including pay, pensions and career progression.
+navigation: 40.1
+layout: "layouts/minimal"
+talk_to_us: true
+backlink: "../"
+breadcrumb: true
```

Comment:
@spencerldixon Do we need this `breadcrumb` line now (once your other PR has been merged)?

Reply:
Good spot! Yes we can get rid of this if the breadcrumb pr is going in first